SPECS-EXTENDED/linuxptp/linuxptp.spec

```diff
@@ -1,72 +1,167 @@
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-%global gitfullver e0580929f451e685d92cd10d80b76f39e9b09a97
-%global gitver %(c=%{gitfullver}; echo ${c:0:6})
 %global _hardened_build 1
-%global testsuite_ver a7f6e1
-%global clknetsim_ver 79ffe4
+%global testsuite_ver ff37e2
+%global clknetsim_ver 9ed48d
+%global selinuxtype targeted
+%bcond_with selinux
 
 Name:		linuxptp
-Version:	2.0
-Release:	8%{?dist}
+Version:	3.1.1
+Release:	7%{?dist}
```

Comment:
This should start with `1`

Reply:
Thanks for catching that.

Reply:
I am having some doubts now about the release starting from 1.
Previously, in linuxptp v2.0, the Release tag to be a newer one than the release upstream.
Therefore, since in the changelogs, the latest upstream release is `3.1.1-6`, I think setting the Release in the spec file to `3.1.1-7` makes sense.
If we set it to `3.1.1-1` it might confuse the reader that this is pointing to the upstream release `3.1.1-1`. 
What do you think?

Reply:
IMO, we always start from 1, once we change the `version`.
Also, the `release` tag is specific to AzureLinux/CBL-Mariner, as the full name of the rpm would have `cm2`.